src/Build/Logging/FancyLogger/FancyLoggerProjectNode.cs

```diff
@@ -68,34 +72,33 @@ public void Log()
                 Console.WindowWidth
             );
             // Create or update line
-            if (Line == null) Line = FancyLoggerBuffer.WriteNewLine(lineContents, false);
-            else FancyLoggerBuffer.UpdateLine(Line.Id, lineContents);
+            if (Line is null) Line = FancyLoggerBuffer.WriteNewLine(lineContents, false);
+            else Line.Text = lineContents;
 
             // For finished projects
             if (Finished)
             {
-                if (CurrentTargetLine != null) FancyLoggerBuffer.DeleteLine(CurrentTargetLine.Id);
+                if (CurrentTargetLine is not null) FancyLoggerBuffer.DeleteLine(CurrentTargetLine.Id);
```

Comment:
You don't have to change this now but please try not to mix this kind of style change into functional PRs. Making them is great, but it delays reviewing the harder changes. If they're in their own PR we can review that quickly and get it in, then focus on the hard stuff.